src/DraftJSEditor.jsx

```diff
@@ -349,13 +351,33 @@ class DraftJSEditor extends Component {
       ]);
     }
 
+    const passedProps = this.props;
+
+    const cleanObject = (object, keyArray) => {
+      let newObj;
+      keyArray.forEach((x) => {
+        const {[x]: deletedKey, ...otherKeys} = object;
+        newObj = otherKeys;
+      })
+      return newObj;
+    }
+
+    const passableProps = cleanObject(passedProps, [
```

Comment:
If I'm understanding what's going on here at a glance, I think something along these lines would accomplish the same?

`const { content, customBlockControls, customBlocks, inlineControls, controlDisplay, containerStyle, ...passableProps } = this.props;`

With the destructuring syntax you're assigning those first 6 things as `const`s and then the rest of the object will get dumped in `passableProps`. Then you might not need that utility function above?